Source/Blazorise.AntDesign/Select.razor.cs

```diff
@@ -168,6 +180,31 @@ static IEnumerable<TValue> ParseMultipleValues( string csv )
             }
         }
 
+        private void SetComparer( IEqualityComparer<TValue> comparer )
+        {
+            var _items = new Dictionary<TValue, RenderFragment>( comparer );
```

Comment:
Maybe just clear member `items` and apply new values directly? Also please remove underscore from var names. tnx